.pre-commit-config.yaml

```diff
@@ -0,0 +1,77 @@
+repos:
```

Comment:
This file puts a standard linter in place for both our locals and the GitHub Action.